pkgs/development/libraries/spandsp/default.nix

```diff
@@ -1,23 +1,24 @@
-{stdenv, fetchurl, audiofile, libtiff}:
+{ stdenv, fetchFromGitHub, audiofile, libtiff, autoreconfHook }:
 stdenv.mkDerivation rec {
-  version = "0.0.6";
+  version = "3.0.0";
   pname = "spandsp";
-  src=fetchurl {
-    url = "https://www.soft-switch.org/downloads/spandsp/spandsp-${version}.tar.gz";
-    sha256 = "0rclrkyspzk575v8fslzjpgp4y2s4x7xk3r55ycvpi4agv33l1fc";
+  src = fetchFromGitHub {
+    owner = "freeswitch";
+    repo = pname;
+    rev = "6ec23e5a7e411a22d59e5678d12c4d2942c4a4b6"; # upstream does not seem to believe in tags
+    sha256 = "03w0s99y3zibi5fnvn8lk92dggfgrr0mz5255745jfbz28b2d5y7";
   };
 
   outputs = [ "out" "dev" ];
 
-  propagatedBuildInputs = [audiofile libtiff];
+  nativeBuildInputs = [ autoreconfHook ];
+  propagatedBuildInputs = [ audiofile libtiff ];
+
   meta = {
     description = "A portable and modular SIP User-Agent with audio and video support";
-    homepage = "http://www.creytiv.com/baresip.html";
+    homepage = "https://github.com/freeswitch/spandsp";
     platforms = with stdenv.lib.platforms; linux;
```

Comment:
```suggestion
    platforms = with stdenv.lib.platforms; unix;
```

Reply:
Have you tested this? I don't really have any way to test this.

Reply:
Just tested macos 10.14, builds successfully.